packages/core-web/asset/css/github.min.css

```diff
@@ -1 +1 @@
-.hljs{display:block;overflow-x:auto;padding:0.5em;color:#333;background:#f8f8f8}.hljs-comment,.hljs-quote{color:#998;font-style:italic}.hljs-keyword,.hljs-selector-tag,.hljs-subst{color:#333;font-weight:bold}.hljs-number,.hljs-literal,.hljs-variable,.hljs-template-variable,.hljs-tag .hljs-attr{color:#008080}.hljs-string,.hljs-doctag{color:#d14}.hljs-title,.hljs-section,.hljs-selector-id{color:#900;font-weight:bold}.hljs-subst{font-weight:normal}.hljs-type,.hljs-class .hljs-title{color:#458;font-weight:bold}.hljs-tag,.hljs-name,.hljs-attribute{color:#000080;font-weight:normal}.hljs-regexp,.hljs-link{color:#009926}.hljs-symbol,.hljs-bullet{color:#990073}.hljs-built_in,.hljs-builtin-name{color:#0086b3}.hljs-meta{color:#999;font-weight:bold}.hljs-deletion{background:#fdd}.hljs-addition{background:#dfd}.hljs-emphasis{font-style:italic}.hljs-strong{font-weight:bold}
\ No newline at end of file
+.hljs{display:block;overflow-x:auto;padding:.5em;color:#333;background:#f8f8ff}.hljs-comment,.hljs-quote{color:#998;font-style:italic}.hljs-keyword,.hljs-selector-tag,.hljs-subst{color:#333;font-weight:700}.hljs-literal,.hljs-number,.hljs-tag .hljs-attr,.hljs-template-variable,.hljs-variable{color:teal}.hljs-doctag,.hljs-string{color:#d14}.hljs-section,.hljs-selector-id,.hljs-title{color:#900;font-weight:700}.hljs-subst{font-weight:400}.hljs-class .hljs-title,.hljs-type{color:#458;font-weight:700}.hljs-attribute,.hljs-name,.hljs-tag{color:navy;font-weight:400}.hljs-link,.hljs-regexp{color:#009926}.hljs-bullet,.hljs-symbol{color:#990073}.hljs-built_in,.hljs-builtin-name{color:#0086b3}.hljs-meta{color:#999;font-weight:700}.hljs-deletion{background:#fdd}.hljs-addition{background:#dfd}.hljs-emphasis{font-style:italic}.hljs-strong{font-weight:700}.hljs>span.highlighted{background:#e6e6fa}.hljs.inline{background:#f8f8f8}.hljs:not(.inline){border:1px solid #c8c8c8;border:1px solid rgba(200,200,200,.3)}.code-block-heading{background:#f2f2ff;color:#8787a5}
```

Comment:
pink highlight is missing:
![Untitled](https://user-images.githubusercontent.com/3306138/94580186-30b91b00-02ac-11eb-8c3c-150b54f2ccd4.png)

 let's rename these files as well https://github.com/MarkBind/markbind/pull/1355#discussion_r492646202, since they contain custom styles now too.

Reply:
wondering if we should separate and format our styles as well
I believe the extra whitespace is hardly an issue over being able to figure out which are our custom additions

Reply:
I have to add a `no-lang` class here to detect this. Okay then.

Reply:
> wondering if we should separate and format our styles as well

I agree, it would be easier to read. How about we move these CSS files into `core-web/src/styles` folder instead, alongside `markbind.css`? Reason is that we have already customized it, so it's not purely a third-party asset anymore, like the ones in the current `asset` folder.

I think if we do that, and (if I'm not wrong) webpack will build all CSS inside that folder, then we can just store the un-minified version of it, and we'll pick and choose which one to include during `copyCoreWebAsset` in `Site/index.js`.

Reply:
> I have to add a `no-lang` class here to detect this. Okay then.

okay 

alternatively you could do `[class="hljs inline"]`, but I don't there's much of an issue in either case

> How about we move these CSS files into core-web/src/styles folder instead, so that it's now alongside markbind.css?

Yes that'd be really nice, your understanding is correct too. (also you get the hot reload package when modifying the styles)

The issue at the moment is that there is no webpack configuration-compilation going at site-building time, which requires some amount of groundwork to implement. (currently its only updated at every release)
I'm ok if you want to take this route, but it'll likely take a substantial amount of time to do; Leaving it as `assets` for the moment may be a more feasible approach

We also have another issue https://github.com/MarkBind/markbind/issues/903 which definitely requires this (configuring webpack compilation at site building time)

Reply:
Hmm, I see. Well, let's put a pin on this and revisit in the future when the webpack configuration is able to set this up.

Pushing the latest changes now.